src/overlays/actors/ovl_Eff_Dust/z_eff_dust.h

```diff
@@ -6,11 +6,24 @@
 
 struct EffDust;
 
+typedef void (*EffDustActionFunc)(struct EffDust *, GlobalContext *);
+
 typedef struct EffDust {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x41C];
+    /* 0x014C */ f32 distanceTraveled[0x40]; // normalized. 0.0f to 1.0f
+    /* 0x024C */ Vec3f initialPositions[0x40]; // array of position of each dust particle.
+    /* 0x054C */ u8 index;
+    /* 0x054D */ u8 life; // only considered if actor.params is 2, 3 or 4.
+    /* 0x054E */ char padding[0x02];
+    /* 0x0550 */ f32 dx; // normalized. 0.0f to 1.0f
+    /* 0x0554 */ f32 dy; // normalized. 0.0f to 1.0f
+    /* 0x0558 */ f32 dz; // normalized. 0.0f to 1.0f
+    /* 0x055C */ f32 scalingFactor;
+    /* 0x0560 */ EffDustActionFunc updateFunc; // function to be called by EffDust_Update()
```

Comment:
```suggestion
    /* 0x0560 */ EffDustActionFunc actionFunc; 
```
We dont really need the comment